### ISSUE TITLE
disable aXe for Migration pages

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly-next#864]
Since the Migration pages pull in examples of PF3 components and since these components might not pass aXe validation, it might be worth disabling aXe for these pages. This will avoid having issues counted that are outside the scope of PF4, and the PF4 components being used on these pages should be going through these checks anyway in their Component pages.

**Comment by jessiehuff** [comment]
It looks like PR #1576 should have closed this issue. I don't think it's relevant with our new testing anyway.
